lib/google/ads/google_ads/interceptors/logging_interceptor.rb

```diff
@@ -302,9 +302,7 @@ def build_summary_message(request, call, method, is_fault)
             customer_id = "N/A"
             customer_id = request.customer_id if request.respond_to?(:customer_id)
             # CustomerService get requests have a different format.
-            if request.respond_to?(:resource_name)
-              customer_id = request.resource_name.split('/').last
-            end
+            customer_id ||= request.resource_name.split('/').last if request.respond_to?(:resource_name)
```

Comment:
I must be honest. I'm not even sure if this line is still required. After digging through the blames, it's existed since V5/6 and if I look at the CustomerService client now, I think the only operation which _doesn't_ set the header is `list_accessible_customers`:

https://github.com/googleads/google-ads-ruby/blob/3dfe04c53f01902f99e080e111a7e65cdb78ea12/lib/google/ads/google_ads/v14/services/customer_service/client.rb#L296

Other operations in that client appear to set the header when available:

https://github.com/googleads/google-ads-ruby/blob/3dfe04c53f01902f99e080e111a7e65cdb78ea12/lib/google/ads/google_ads/v14/services/customer_service/client.rb#L227-L230

The other aspect is I'm not even sure that `list_accessible_customers` responds to `resource_name`, so the line might not even trigger for this scenario (how could it, it returns a list)?